x/ccv/provider/keeper/relay.go

```diff
@@ -376,12 +372,12 @@ func (k Keeper) HandleSlashPacket(ctx sdk.Context, chainID string, data ccv.Slas
 		"infractionType", data.Infraction,
 	)
 
-	// Obtain provider chain consensus address from packet data
-	// (overwritten with proper provider chain cons address in OnRecvSlashPacket)
-	providerConsAddr := sdk.ConsAddress(data.Validator.Address)
+	consumerConsAddr := sdk.ConsAddress(data.Validator.Address)
```

Comment:
The log just above should print `providerConsAddr` no?

Reply:
Agree

Reply:
Move it down to where you have providerConsAddr and then print both prvoiderConsAddr and ConsumerConsAddr in the log

Reply:
Done